umbraco-cms-webapp-redis-cache/azuredeploy.json

```diff
@@ -351,150 +392,144 @@
     },
     {
-      "apiVersion": "2014-04-01",
       "name": "[concat('ServerErrors ', parameters('appName'))]",
-      "type": "microsoft.insights/alertrules",
-      "location": "[variables('appInsightsRegion')]",
-      "tags": {
-        "[concat('hidden-link:', resourceGroup().id, '/providers/Microsoft.Web/sites/', parameters('appName'))]": "Resource"
-      },
+      "type": "Microsoft.Insights/metricAlerts",
+      "apiVersion": "2018-03-01",
+      "location": "global",
       "dependsOn": [
         "[concat('Microsoft.Web/sites/', parameters('appName'))]"
       ],
-      "properties": {
-        "name": "[concat('ServerErrors ', parameters('appName'))]",
-        "description": "[concat(parameters('appName'), ' has some server errors, status code 5xx.')]",
-        "isEnabled": false,
-        "condition": {
-          "odata.type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.ThresholdRuleCondition",
-          "dataSource": {
-            "odata.type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.RuleMetricDataSource",
-            "resourceUri": "[concat(resourceGroup().id, '/providers/Microsoft.Web/sites/', parameters('appName'))]",
-            "metricName": "Http5xx"
-          },
-          "operator": "GreaterThan",
-          "threshold": 0,
-          "windowSize": "PT5M"
-        },
-        "action": {
-          "odata.type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.RuleEmailAction",
-          "sendToServiceOwners": true,
-          "customEmails": []
-        }
-      }
-    },
-    {
-      "apiVersion": "2014-04-01",
-      "name": "[concat('ForbiddenRequests ', parameters('appName'))]",
-      "type": "microsoft.insights/alertrules",
-      "location": "[variables('appInsightsRegion')]",
       "tags": {
         "[concat('hidden-link:', resourceGroup().id, '/providers/Microsoft.Web/sites/', parameters('appName'))]": "Resource"
       },
-      "dependsOn": [
-        "[concat('Microsoft.Web/sites/', parameters('appName'))]"
-      ],
       "properties": {
-        "name": "[concat('ForbiddenRequests ', parameters('appName'))]",
-        "description": "[concat(parameters('appName'), ' has some requests that are forbidden, status code 403.')]",
-        "isEnabled": false,
-        "condition": {
-          "odata.type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.ThresholdRuleCondition",
-          "dataSource": {
-            "odata.type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.RuleMetricDataSource",
-            "resourceUri": "[concat(resourceGroup().id, '/providers/Microsoft.Web/sites/', parameters('appName'))]",
-            "metricName": "Http403"
+        "severity": 1,
+        "enabled": true,
+        "scopes": [
+          "[resourceId('Microsoft.Web/sites/', parameters('appName'))]"
+        ],
+        "targetResourceType" : "Microsoft.Web/sites",
+        "targetResourceRegion": "[resourceGroup().location]",
```

Comment:
this is likely to cause issues if the defaultValue for the location property is not used...  should be parameter('location')